lib/modules/manager/gomod/readme.md

```diff
@@ -3,6 +3,7 @@ You might be interested in the following `postUpdateOptions`:
 1. `gomodTidy` - if you'd like Renovate to run `go mod tidy` after every update before raising the PR.
    1. This is implicitly enabled for major updates if the user has enabled the option `gomodUpdateImportPaths`
 1. `gomodTidy1.17` - if you'd like Renovate to run `go mod tidy -compat=1.17` after every update before raising the PR.
+1. `gomodTidyE` - if you'd like Renovate to run `go mod tidy -e` after every update before raising the PR.
 1. `gomodUpdateImportPaths` - if you'd like Renovate to update your source import paths on major updates before raising the PR.
 1. `gomodMassage` - to enable massaging of all `replace` statements prior to running `go` so that they will be ignored.
```

Comment:
```suggestion
1. `gomodTidy` - if you'd like Renovate to run `go mod tidy` after every update before raising the PR
   1. This is implicitly enabled for major updates if the user has enabled the option `gomodUpdateImportPaths`
1. `gomodTidy1.17` - if you'd like Renovate to run `go mod tidy -compat=1.17` after every update before raising the PR
1. `gomodTidyE` - if you'd like Renovate to run `go mod tidy -e` after every update before raising the PR
1. `gomodUpdateImportPaths` - if you'd like Renovate to update your source import paths on major updates before raising the PR
1. `gomodMassage` - to enable massaging of all `replace` statements prior to running `go` so that they will be ignored
```

Reply:
Removed all, also on existing list elements.